src/Azs.ContainerRegistry.Admin/test/Remove-AzsContainerRegistryConfiguration.Tests.ps1

```diff
@@ -17,10 +17,10 @@ Describe 'Remove-AzsContainerRegistryConfiguration' {
 
         $maxCapacityInGib = (Get-AzsContainerRegistryConfiguration ).MaximumCapacityInGib
         $targetCapacityInGib = 0
-        Remove-AzsContainerRegistryConfiguration -ConfigurationName default
+        Remove-AzsContainerRegistryConfiguration
```

Comment:
Do you have to rerecord the test for this change?

Reply:
The parameter is optional, so it also works if it is presented.